### DISCUSSION TITLE
Virtual DJ

### DISCUSSION THREAD

**Discussion by Dasher-The-Viral** [VolcanicArts/VRCOSC#133] [Q/A] [answered]
Any way I could integrate Virtual DJ's "Currently Playing" into the textbox commands?

Something like:

Deck 1: Song Title
Deck 2: Song Title
etc.

This would be a very niche, but neat, feature to have! :D

**Comment by VolcanicArts** [comment, marked as answer]
From a quick glance at Google this seems possible with a VDJ plugin to expose that data over a local server. I'll look into it

**Reply by Dasher-The-Viral** [reply]
Thank you so much! You're awesome! ^~^

**Reply by Dasher-The-Viral** [reply]
Just checking to see if there are any updates on this feature. :)

**Reply by VolcanicArts** [reply]
Unfortunately not. Been working on some other things and travelling
